content/_data/books/Learning_Continuous_Integration_with_Jenkins.yml

```
@@ -0,0 +1,21 @@
+title: "Learning Continuous Integration with Jenkins"
+url: https://www.amazon.com/Learning-Continuous-Integration-Jenkins-implementing-ebook/dp/B0751J4L41/
+author: "Nikhil Pathania"
+publication_year: 2017
+publisher: "Packt Publishing Limited"
+type: Paid
+image: books/continous-integration-with-Jenkins.jpg
+description: |-
+  Speed up the software delivery process and software productivity using the latest features of Jenkins 
+  
+  . Take advantage of a Continuous Integration and Continuous Delivery solution to speed up productivity and achieve faster software delivery 
+  . See all the new features introduced in Jenkins 2.x, such as Pipeline as code, Multibranch pipeline, Docker Plugin, and more 
+  . Learn to implement Continuous Integration and Continuous Delivery by orchestrating multiple DevOps tools using Jenkins 
+  . Get to know some of the most popular ways to set up Jenkins 
+  . Manage users, projects, and permissions in Jenkins to ensure better security
+  . Leverage the power of plugins in Jenkins 
+  . Learn how to create a CI pipeline using Jenkins Blue Ocean 
+  . Create a distributed build farm using Docker and use it with Jenkins 
+  . Implement CI and CD using Jenkins 
+  . See the difference between CD and Continuous Deployment 
+  . Understand the concepts of CI In Detail. 
```

```suggestion
In this book, you'll discover how to take advantage of a Continuous Integration and Continuous Delivery solution to speed up productivity and achieve faster software delivery.
You'll read about all the new features introduced in Jenkins 2.x, such as Pipeline as code, Multibranch pipeline, the Docker Plugin, and more.
You'll learn how to implement Continuous Integration and Continuous Delivery, by orchestrating multiple DevOps tools using Jenkins.
You'll learn how to manage users, projects, and permissions in Jenkins to ensure better security.
Finally, you'll learn how to leverage the power of plugins in Jenkins!
```
Similar suggestions as above, with making the description shorter and removing some unnecessary sentences/wordings. Updated others to match syntax and single sentence per line formatting.

This single sentence per line formatting will render in-line on the page, so there is no worry about spacing.